cmd/root.go

```diff
@@ -115,11 +116,25 @@ func aliasNormalizeFunc(f *pflag.FlagSet, name string) pflag.NormalizedName {
 }
 
 func initClient() {
+
 	endpoint := authViper.GetString(ckeyAuthEndpoint)
 	if endpoint == "" {
 		return
 	}
 
+	if strings.Contains(endpoint, "fyde") {
+		authViper.Set(ckeyAuthEndpoint, DefaultEndpoint)
+		global.WriteFiles = true
+		if global.WriteFiles {
+			err := authViper.WriteConfig()
+			if err != nil {
+				fmt.Printf("Error writing auth file: %v\n", err)
+				return
+			}
+		}
+		fmt.Fprintf(os.Stderr, "WARNING: The endpoint in use is no longer valid and was replaced by the new default endpoint '%s'. If necessary, please update your endpoint with the 'cluster' command before logging in.\n", DefaultEndpoint)
```

Comment:
```suggestion
		fmt.Fprintf(os.Stderr, "WARNING: The endpoint in use is deprecated and may stop working in the future. Please update your endpoint using 'access-cli cluster set us|eu' command before logging in.\n")
```
@tse4a i suggest we add to the warning the endpoint shortcuts to minimize mistakes while transitioning.

Reply:
This warning is now obsolete. Updating the config automatically to point to respective new endpoint.

Reply:
@oNaiPs updated PR to automatically update existing endpoints.